src/Commands/Reset/PurgeCommand.php

```diff
@@ -91,7 +92,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
     $progress->setMessage('(Re-) Creating Database; executing migrations');
     CommandHelper::executeShellCommand(
-      ['bin/console', 'doctrine:migrations:migrate'], [], 'Execute the migration to the latest version', $output
+      ['bin/console', 'doctrine:migrations:migrate', '--no-interaction'], ['timeout' => 320],
+      'Execute the migration to the latest version', $output
     );
```

Comment:
`'timeout' => 320` seems a bit arbitrary since it would be 5 mins and something. Maybe we can decrease that with a working docker image now!